data-share/data-share-service/src/main/java/io/mosip/datashare/service/impl/DataShareServiceImpl.java

```diff
@@ -138,10 +141,17 @@ public DataShare createDataShare(String policyId, String subscriberId, Multipart
 			String randomShareKey;
 			try {
 				byte[] fileData = file.getBytes();
-				PolicyResponseDto policyDetailResponse = policyUtil.getPolicyDetail(policyId, subscriberId);
-
-
-				DataShareDto dataSharePolicy = policyDetailResponse.getPolicies().getDataSharePolicies();
+				DataShareDto dataSharePolicy;
+				LocalDateTime publishDate = null;

```

Comment:
for static policy it will be null?

Reply:
policyId will be null and have tested the same and it is not causing any issue.

Reply:
So assuming publishDate as null for static policy